### DISCUSSION TITLE
2012/05/15/%E5%86%85%E5%B5%8C%E5%AD%90vi

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#354] [Announcements]
# 2012/05/15/%E5%86%85%E5%B5%8C%E5%AD%90vi

https://labview.qizhen.xyz/zheng-wen-mu-lu/optimization/optimization\inline\subvi

https://blog.qizhen.xyz/2012/05/15/%E5%86%85%E5%B5%8C%E5%AD%90vi

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Shiang
2012年06月11日 13:43 编辑
阮大侠:
小弟有个FPGA相关问题~不知能否请教一下!能互留email吗?
xdainc@gmail.com
感谢

回复
ruanqizhen
2012年06月12日 13:52 编辑
我的Email是 ruanqizhen@gmail。com
不过我对FPGA实在不怎么了解，不知道是否帮得上忙。

回复
请教个移植的问题。
2012年05月24日 03:40 编辑
你好，我现在在做一个移植的工作，就是将6.1中的labvIEW程序，移植到高等级的版本，比如说8.5中，但是这个程序不是我编写的，我对labvIEW也不熟悉，所以我可能不想详细的了解其中的控制方法，只是要移植本身，但是其中，低版本的很多子VI,要如何导出并且移植到高版本中呢，谢谢

回复
ruanqizhen
2012年05月24日 10:49 编辑
老的项目在新版LabVIEW中打开，一般不需要特殊的移植工作。可以直接在高版本LabVIEW中打开并使用的。

回复
请教个移植的问题。
2012年05月25日 03:36 编辑
好的 非常感谢，有问题再请教您。

回复
